### PR TITLE
Gael trial

### PR DESCRIPTION
**Pull request by elgael06**

### CODE REVIEW

File: web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala [hunk 1]
```diff
@@ -129,27 +135,30 @@ import scala.util.{Failure, Success}
         .disabled(loading)
         .variant(muiStrings.contained)
         .color(Color.primary)
-        .onClick(_ => doLogin())
+        .`type`(muiStrings.submit)
+        //.onClick(_ => doLogin())
     }
 
     // TODO: Use a form to get the enter key submitting the form
```
Comment (AlexITC) [inline web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala:142]: This comment does not apply anymore, remove it

File: web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala [hunk 2]
```diff
@@ -129,27 +135,30 @@ import scala.util.{Failure, Success}
         .disabled(loading)
         .variant(muiStrings.contained)
         .color(Color.primary)
-        .onClick(_ => doLogin())
+        .`type`(muiStrings.submit)
+        //.onClick(_ => doLogin())
```
Comment (AlexITC) [inline web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala:139]: Remove unused code

File: web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala [hunk 3]
```diff
@@ -129,27 +135,30 @@ import scala.util.{Failure, Success}
...
     }
 
     // TODO: Use a form to get the enter key submitting the form
+    form(onSubmit := (doLogin(_)))(
     mui
```
Comment (AlexITC) [inline web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala:144]: This seems to have wrong alignment, which reminds me to enable scalafmt check in circleci

**Review by AlexITC — CHANGES_REQUESTED** [review]
Overall, looks good, just remove the useless files (`.metals/*`/`*.iml`)

**Comment by elgael06** [comment]
Use form: submit event  enter key

**Comment by elgael06** [comment]
remove the useless files (.metals/*/*.iml)

**Review by AlexITC — APPROVED** [review]

**Pull request merged**